lib/matplotlib/animation.py

```diff
@@ -468,14 +469,15 @@ def grab_frame(self, **savefig_kwargs):
     def finish(self):
         # Call run here now that all frame grabbing is done. All temp files
         # are available to be assembled.
-        self._run()
-        super().finish()  # Will call clean-up
-
-    def _cleanup(self):  # Inline to finish() once cleanup() is removed.
```

Comment:
This is supposed to be inlined once `cleanup` is gone, but I guess it's not yet? So inlining might break an explicit `cleanup` call?

Reply:
`def cleanup` is gone, it went in e1eca0aa8bf0b51009e012cd37d3e95f364d0ee9 / https://github.com/matplotlib/matplotlib/pull/22516  .

That is going to make the backport annoying (but maybe we can just drop this part of it)?

Reply:
Luckily the test suite worked like it was supposed to and we caught it before it went out 😄 .